xlml/utils/metric.py

```diff
@@ -252,6 +252,14 @@ def process_tensorboard_summary(
   metric_history_rows = []
   metadata_history_rows = []
 
+  metadata_history_rows.append(
```

Comment:
I think this is more appropriate to add into `add_test_config_metadata()` method. You may need to check if this included in `MetricConfig`.

Reply:
Done!